crates/networking/rpc/authentication.rs

```diff
@@ -1,4 +1,4 @@
-use crate::RpcErr;
+use crate::utils::RpcErr;
```

Comment:
I'm not a big fan of `utils` modules with a mix of unrelated functions. Can't we have an `errors` module for rpc, just like we have in some other crates?

Reply:
Yes, let me tackle that in a follow up.